packages/node-cli/README.md

```diff
@@ -12,7 +12,7 @@ CLI for Node.js is an interactive tool that guides you through all the steps of
 
 ## Prerequisites
 The following software is required to work with SuiteCloud CLI for Node.js:
-- Node.js version 12.14.0 LTS or greater
+- Node.js version 14 LTS or greater
```

Comment:
@vic-nik Is it safe to say use v14 or greater? New node versions might cause problems in installation etc.